pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -624,10 +681,14 @@ func selectVictimsOnNode(
 	// support this case for performance reasons. Having affinity to lower
 	// priority pods is not a recommended configuration anyway.
 	if status := fh.RunFilterPluginsWithNominatedPods(ctx, state, pod, nodeInfo); !status.IsSuccess() {
+		// `framework.Error` will not be included in the status map, this align with main scheduler process, log the warning
+		// message here and continue to check other `potentialNodes`.
```

Comment:
since we are not breaking on error like we do in generic_scheduler.go, I think we should add it to the map.

Reply:
in any case, this function should simply return status.

Reply:
Done